templates/partials/popupmenu.handlebars

```diff
@@ -0,0 +1,48 @@
+<div class="modal fade popupmenu" id="popupMenu" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="exampleModalLabel">
+                    <a href="http://vlab.co.in" class="navbar-brand">
+                        <img src="{{assets_path}}/assets/images/vlabs-color-small-moe.jpg" alt="vlabs logo" class="vlabs-logo">
+                    </a>
+                </h5>
+                <button type="button" class="btn-close align-self-center" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body nav-menu">
```

Comment:
Refactor code to take the component out and import it into both popup menu and sidebar.